locals.tf

```diff
@@ -0,0 +1,13 @@
+locals {
+  default_tags = {
+    "Business Unit" = "OIT",
```

Comment:
Let's swap these out for something more generic

Reply:
done